config/locales/en.yml

```
@@ -2241,6 +2242,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   price_sack: "Price Sack"
   new_order_cycles: "New Order Cycles"
   new_order_cycle: "New Order Cycle"
+  new_order_cycle_tooltip: "Fulfil orders for a time perioid"
```

Great work on learning how the software works. I have a different suggestion here though:
```suggestion
  new_order_cycle_tooltip: "Open shop for a certain time period"
```